### PR TITLE
Chapter 14 following users

### PR DESCRIPTION
**Pull request by Arizar98**

### CODE REVIEW

**Comment by Arizar98** [comment]
Ready

File: app/assets/stylesheets/relationships.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Relationships controller here.
```
Comment (manhnt-0960) [inline app/assets/stylesheets/relationships.scss]: delete this empty file

File: app/helpers/relationships_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module RelationshipsHelper
```
Comment (manhnt-0960) [inline app/helpers/relationships_helper.rb]: similar

File: app/models/relationship.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class Relationship < ApplicationRecord
+  belongs_to :follower, class_name: "User"
```
Comment (manhnt-0960) [inline app/models/relationship.rb]:
```suggestion
  belongs_to :follower, class_name: User.class
```

File: app/models/user.rb [hunk 4]
```diff
@@ -1,6 +1,11 @@
 class User < ApplicationRecord
 
   has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name: "Relationship", foreign_key: "follower_id", dependent: :destroy
```
Comment (manhnt-0960) [inline app/models/user.rb]:
```suggestion
  has_many :active_relationships, class_name: Relationship.class, foreign_key: :follower_id, dependent: :destroy
```

**Comment by Arizar98** [comment]
Ready

**Comment by manhnt-0960** [comment]
merged

**Pull request merged**